src/core/observables/PidObservable.hpp

```diff
@@ -51,7 +51,7 @@ using ParticleReferenceRange =
  */
 class PidObservable : virtual public Observable {
   /** Identifiers of particles measured by this observable */
-  std::vector<int> m_ids;
+  const std::vector<int> m_ids;
```

Comment:
I think, as a general rule, const data members should be avoided.

Reply:
ok, changed that